attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -104,19 +111,20 @@ account.multisigSetDefaults@return = ()
 ; tag::submitTransaction[]
 ; Submit a transaction to be executed when the threshold is reached.
 account.multisigSubmitTransaction@param = {
-    ; The account to submit the transaction. If omitted, the content of the
+    ; The account to submit the transaction to. If omitted, the content of the
     ; transaction itself may be read to figure this out, or the server MAY
     ; return an error.
     ? 0 => non-anonymous-identity,
 
-    ; A memo associated with the transaction.
-    ? 1 => bstr,
+    ; A memo associated with the transaction. This is meant to be human readable
+    ; and can be used to explain the transaction to approvers.
+    ? 1 => memo,
 
     ; The transaction to send. Must be a transaction ledger info.
     ; The _from_ field (or source) of the transaction will be used to resolve
     ; the account.
     ; Servers are allowed to error on an invalid or unsupported transaction.
-    2 => account-multisig-transaction,
+    2 => transaction-ledger-info,
```

Comment:
`transaction-ledger-info` doesn't exist anywhere. Did you mean `ledger-transaction-info`? And if so, why change it from `account-multisig-transaction = ledger-transaction-info`?

Reply:
How did this pass CI then? We type check the CDDL...

Reply:
Apparently, the CDDL check doesn't validate this... Urgh.